geoportal/c2cgeoportal_geoportal/scaffolds/create/redis/redis.conf.tmpl

```diff
@@ -0,0 +1,2 @@
+maxmemory ${REDIS_MAXMEMORY}
+maxmemory-policy allkeys-lru
```

Comment:
You can pass that from the command line: https://redis.io/topics/config

And I recommend adding: `--save "" --appendonly no`

Reply:
https://github.com/camptocamp/private-geo-charts/blob/177b618cc5ba9f7fab8e6209fd2ae204442dcc97/c2cgeoportal/templates/redis-deployment.yaml#L34-L42

Reply:
Thanks, fixed :-)